opsramp/devmgmt.py

```diff
@@ -41,3 +41,46 @@ def run(self, uuid):
 
     def delete(self, uuid):
         return self.api.delete('%s' % uuid)
+
+
+class Discovery(ApiWrapper):
+    def __init__(self, parent):
+        super(Discovery, self).__init__(parent.api, 'policies/discovery')
+
+    def search(self, pattern=''):
+        return self.api.get('/search?name=%s' % pattern)
+
+    def get(self, discoveryProfileId=''):
+        return self.api.get('/%s' % discoveryProfileId)
+
+    def create(self, definition):
+        return self.api.post('', json=definition)
+
+    def update(self, definition):
```

Comment:
in my tests, definition needs to include "id" even though the docs don't actually say that :) another assert.